.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "web"]
+	path = rohd
+	url = https://github.com/quekyj/rohd
```

Comment:
we should update this to the main repo and branch before merging.  do you want to merge in the changes into the main repo first?

Reply:
Ya, we can do that